packages/example-getting-started/docs/7-controller.md

```diff
@@ -5,7 +5,7 @@ Now, we'll create a controller to handle our Todo routes. Create the
 - `index.ts` (export helper)
 - `todo.controller.ts`
 
```

Comment:
Quick explanation of what a controller's role is and a link to it's docs page would be good here.